example/generator/odp_generator.c

```diff
@@ -25,6 +25,8 @@
 #define MAX_WORKERS            (ODP_THREAD_COUNT_MAX - 1)
 #define POOL_NUM_PKT           2048  /* Number of packets in packet pool */
 #define POOL_PKT_LEN           1856  /* Max packet length */
+#define MAX_HDR_LEN            128   /* L2/L3/L4 header length */
```

Comment:
The 128 is a magic number because is cannot be freely selected but depends on other things. It seems it should be
(ODPH_ETHHDR_LEN + ODPH_IPV4HDR_LEN + MAX(ODPH_UDPHDR_LEN, ODPH_ICMPHDR_LEN))

Also, the comment can be interpreted as "L2 or L3 or L3 header length". Replacing the slash with e.g. a plus sign would make it better.

Reply:
ACK